e2e/nsenter.go

```diff
@@ -0,0 +1,257 @@
zVDV2I2RlAxR1ExTlk2Wmk4CjZqWTlTMTkwWmUwM2x5Q1FJK1VjMjNXb0FCajJaRGtEUU1vcWNYc3J0cUlyQmwzdHFGTlNHU1c4czMrbFVCaTMzTzBQaUwKUG1pdDNPNG5kT0d3UGpsODUxdGMyQXVVdVFCWGNzdGJBVFRJTWd1eUYvcUpLTzNuTTFuVTFSOTR4YTNFR2pCL3hYOFRLRQp6emx3WnZNTCtzVkZhYkNpaWQybVpjN0JyQytMbEVSZndkcU1NbmFlNnNlNzU1cW9wdGlQRm1hQzNUR3IzQzlMZFpZRWZLCm50MVBYQitLcDJxL1AzRnVuMEhiN2dXc3dkLy85MGJYODhlcXlhRnRBQUFGbU5JSHloZlNCOG9YQUFBQUIzTnphQzF5YzIKRUFBQUdCQUxVUVJOTkRqdEQxV0VBQzJNSnZ0K055STlUWDRsNkdpQjFjS3ZvZUV4WWN0WVp5S0lLQ0pMUGtLWVlFRnhWSQpjbGNvRm85MnJVSndZWHlwT0kvc0x2QXpLajViQjU0Tld1WkxPK2xBcjlFRTJ2c2s3MVM0RGlpQzdFSVl5MUdoejBDVGd3ClR5SHpUNmhLUWQyeUNMRnR6SjVkU25OMkU3VngrSjIxWmJON05ZZ29oRE1qRG5DQktHY1hEaU9yVkdjQTBmUUxQeHBvTjAKYmxKazZMbjg0S2h1WlIwdTVBeUsrcXpyNDhEWGY2TG9CbTF3MVBQdjJrdVFsbStoVDlSa05UV09tWXZPbzJQVXRmZEdYdApONWNna0NQbEhOdDFxQUFZOW1RNUEwREtLbkY3SzdhaUt3WmQ3YWhUVWhrbHZMTi9wVkFZdDl6dEQ0aXo1b3JkenVKM1RoCnNENDVmT2RiWE5nTGxMa0FWM0xMV3dFMHlESUxzaGY2aVNqdDV6TloxTlVmZU1XdHhCb3dmOFYvRXloTTg1Y0diekMvckYKUldtd29vbmRwbVhPd2F3dmk1UkVYOEhhakRKMm51ckh1K2VhcUtiWWp4Wm1ndDB4cTl3dlMzV1dCSHlwN2RUMXdmaXFkcQp2ejl4YnA5QjIrNEZyTUhmLy9kRzEvUEhxc21oYlFBQUFBTUJBQUVBQUFHQkFJVWlra2o1cXdEUTluUVM0OG1NbGgxQzV6Ci94QWIxWmxLcHFCQnZuazBjMkV1L3A3d21Qb21jNEJLUkxTNkhWcEdXYno0THIwNWcveEI4QzJ0bFE1RzZ3WUlaN0xzMnoKcWp3ZDQ4NnVSdGtkaGRzWEhIZ2g0aDg5clhVb2dBL2xOOXlXMnNiL05aMGgrL2dsRCtRTkFlR1UvMDd2S1pSd0txN0JFaQp1Y1pRZVZ0RjEzYzJkcjZZNmRscnBYVHEzT1BrUVh6OTdBZUJ5K3R3UVV1TjlSSnV3U1NIMDk1bWVtcFk0UzRXUUIwUk03CklzYkVLWUc0d3IyL2NpNWc0YThkZ3BsRnIzUHlGb2tVRTFOQzExZDYvRlMrL2c3U05GOXBNWXE0b2RHZHkrKzJDRmc0S1IKa1duOHdVb0paV2h3YzNnQ0RlYnhBZmdWd3V6NXE5WldWMW1SRWt0VmhUZGY3dGJQeU92YjM2NjU1eWMycUhON1Z3T3h5ZwpERFJPWUkvaEVlUUcwdFFTQ2NxZHlqczFnVU9lL2owNUZCUFkzaXh3Qzh4RG5YVXVPOURjbG9RNVptT3JmRUhhRW8yYmRSCk1rNWNraGJTd21uNlU5Qmp3Q04rbENxK1hrT1d6L2ZOcms3Mm1IY2FQTWt4WmxKelpCNGNBOTBkaHg3YXpQd0ZmS1FRQUEKQU1BdklPZk8wMVVpRjdPS05xSHdaV0txU2VkVVRUa2hqd2RzdmZ5bzg0bDViQWZDVTVRRVNkbzdBOGlSZUhpUW5ZckQ5ZwpuR2IrNEJrZDI4S1E1UkxOb1lxZDgrK2xhQWUyNE1SVFFPc3psUS9LK3RRWWdWazBOaTRjeFhsWFc1TzVyTEFTbEw3THQ5CmorbzJiQkluSWs2TThZYStnRFJrWWtmYXhIWENoK0FxOUxXNEF4bHhJZEc2SzRQOXlYcUhGakNyMVRvNGk2NUtHUkQ1Wm4KNDFVR1paVUpWakZBL0E3SUM2KzVlSDZQSkE4MUk3VER2Tmt0WGQ0VGhQRTd5S2lCMEFBQURCQU9VWGU2b2JGWk13NjFMUgozVE84NFd4Q09TUjgvamNIRDNxSHRKdFdMakkvYk53MGthQVhOMUpsdHFiTElBL08ydFdPY2ZEeEpFYU1sQjRUcmdZV0FUCnI1VkZFV1hZS0MySGxSVGEva2c3QS9HUk9Od1hIS2V5dXlscFVNN0N6MTdMa2k5ZTF5QkVOYTRGQmg4V3c3VGtjaGtrVDcKc3dsbEh3QlI3dGlTWlBEZXl2UzZqZUo3OWxwNlFGMTc3NTZmdHg1RjAxbDZVV3hUYmlPSndiU3FxZUxQS1E5SldnRWRTQwp2MjNzcTNQV1F2Q1ZiSWtkOS9YK1piWVFCTzhOUWYvUUFBQU1FQXlsU2lvclF4dDlQY2ljTy9taXJjdlpzTC9tdldORy95ClY1L1BoRml3WTJ2c0d1RUFSYkxFWEQvSkVDck5rM3VZS0VZRnFKZXRIbTl6ajdlcWlJSHFWemtHaU13ZE9zNXZQM25sREsKK05PWjFiNG9rVzgwdjRJWWduZ2pqN1B5a2kreUxMQzlnZS9lQ3NhM1pvNDNMUTNoQXNhcFZ6N3YxUnYvMFAzcGVteW8zdgo5K0lUUlVTdnpuaTd2bGIxK0JIMzBMdTNOWm9pWmZ4K0JpMHJFU1lBN29qaXFjK21SZG5BT3Q3UWw0QlJ4N1BZRWkyWkdKCk8xZ0JLWnY2OFRzQ294QUFBQUlYSnZiM1JBVldKMWJuUjFMVEl3TURRdFptOWpZV3d0TmpRdGJXbHVhVzFoYkFFPQotLS0tLUVORCBPUEVOU1NIIFBSSVZBVEUgS0VZLS0tLS0K`
```

Comment:
TODO: remove this and use the generated key again

log the generated key somewhere?